CONTRIBUTING.md

```diff
@@ -36,7 +36,12 @@ you must run a local docker daemon.
 check formatting:
 
     `./gradlew build`
+    
+4. If you are a Windows user, use the alternate command mentioned below to run tests and
```

Comment:
I think these changes are already on the `main` branch, so you probably should rebase your branch against the upstream/main to have a clean PR